downstream/modules/platform/ref-controller-internal-cluster-routing.adoc

```diff
@@ -2,23 +2,16 @@
 
 = Internal Cluster Routing
 
-The {ControllerName} cluster hosts communicate across the network within the cluster. 
-In the inventory file for the traditional VM installer, multiple routes are indicated to the cluster nodes that are used in different ways:
+{ControllerName} cluster hosts communicate across the network within the cluster. In the inventory file for the traditional VM installer, you can indicate multiple routes to the cluster nodes that will be used in different ways:
```

Comment:
Is it 'the' {ControllerName} something, or just {ControllerName} something. We've tended to use 'the'.